src/render/systems.rs

```diff
@@ -97,9 +98,6 @@ pub fn create_collider_renders_system(
                         ground_pbr.draw,
                         ground_pbr.render_pipelines,
                         ground_pbr.transform,
-                        ground_pbr.translation,
-                        ground_pbr.rotation,
-                        NonUniformScale(scale),
```

Comment:
I'm not certain if there's some additional component that's supposed to be added to indicate that there's a non-uniform scale that should be looked at.